storage/engine/mvcc.go

```diff
@@ -1236,7 +1238,12 @@ func MVCCFindSplitKey(engine Engine, raftID int64, key, endKey proto.Key) (proto
 		done := !bestSplitKey.Equal(encStartKey) && diff > bestSplitDiff
 
 		// Add this key/value to the size scanned so far.
-		sizeSoFar += int64(len(kv.Key) + len(kv.Value))
+		_, _, isValue := MVCCDecodeKey(kv.Key)
+		if isValue {
```

Comment:
The name `isValue` was mildly confusing to me as it is indicating whether a timestamp was present. Perhaps rename to `hasTimestamp`.